src/common/proc_args.c

```diff
@@ -589,6 +589,11 @@ char * base_name(char* command)
 	return name;
 }
 
+/* use_gbytes:
+   1 : no unit is GB
+   2 : no unit is B
+   anything else : no unit is MB
+*/
 static long _str_to_mbtyes(const char *arg, int use_gbytes)
 {
 	long result;
```

Comment:
Should be `uint64_t`. sizeof(long) >= sizeof(int) which might still be 32 and not 64. If you are going to allow passing in bytes, you really want to avoid overflow as 185GiB > 2^31-1.

Reply:
the error is already caught

Reply:
Still depends on the sizeof(long) on your machine.